projectDocs/translating/readme.md

```diff
@@ -0,0 +1,55 @@
+# NVDA Translation and Localization
+
+Thank you for considering to localize NVDA to your language, or improving an existing translation.
+In order to support multiple languages/locales, NVDA must be translated and data specific to the locale must be provided.
+There are several tasks to be done, and this document hopes to give you an overview and guide you through the process.
+
+## Translation Mailing List
+Translators should subscribe to the [NVDA translations mailing list](https://groups.io/g/nvda-translations) hosted at Groups.IO.
+
+It is an English low traffic list devoted for the discussion of translation. 
+Important messages that relate to translators will also be sent here, e.g. before official NVDA releases, to remind translators to make sure their localization is up to date.
+
+## Important Dates
+
+Translators should ensure their translation is up to date during the beta period before each new release, in order for it to be included in the upcoming final release.
+A reminder will be posted on the translation mailing list giving two weeks notice of the translation string freeze.
+Work submitted after the translation string freeze date will not be included in the upcoming release.
+For further information please see the [Release Process page](https://github.com/nvaccess/nvda/blob/master/projectDocs/community/releaseProcess.md).
+
+## Translation Status
+
+You can view [Crowdin](https://crowdin.com/project/nvda) for an up to date report on the status of translating the NVDA interface.
+If you would like to improve or would like to work on a new language, please write to the [NVDA translations mailing list](https://groups.io/g/nvda-translations).

```

Comment:
@michaelDCurran is this more accurate?

```suggestion
If you would like to improve an existing language, please write to NV Access via info@nvaccess.org.
```

Reply:
I think I would prefer to keep this on the translations list. There are more people that can help.  Perhaps we can give a couple of trusted community members full language coordinator permissions on Crowdin for all languages, thus they will be able to assign permissions for others. As for the translation system itself: we will not be taking any new translations for documentation on that system. It will be moving to po files on Crowdin  sometime this year. If I miss anything on the translation list for now,  trust that a community member will nudge me :)

Reply:
Recently, I have had to remind NV Access of pending requests of people wanting to translate new language, hence my proposal to contact you directly.

If you prefer delegating the possibility to create new languages and grant approve rights to trusted members, that's another path. And I am completely fine with that.